rts/Sim/MoveTypes/GroundMoveType.cpp

```diff
@@ -635,23 +638,25 @@ void CGroundMoveType::UpdatePreCollisions()
 	oldSpeed = newSpeed = 0.f;
 }
 
-void CGroundMoveType::UpdateUnitPositionAndHeading() {
+void CGroundMoveType::UpdateUnitPosition() {
+	resultantForces *= 0.f;
```

Comment:
why the multiplication and not just `= 0.f`?

Reply:
Because it is a `float3`. However, you right that it is better to `= ZeroVector`.